blog/views.py

```diff
@@ -68,7 +68,7 @@ def article_edit(id):
         article.title = request.form.get('title')
         article.content = request.form.get('content')
         if bool(request.form.get('published')):
```

Comment:
Editing the publication date is probably not the best idea of the world.
The issues says we should, but the need is to preserve the initial publication date.
I would therefore only set a publication date when it has none yet.
Therefore, an article that has a publication date should never be able to change it; it's it initial publication date:

```python
if bool(request.form.get('published') and not  article.publication_date):
    article.publication_date = datetime.datetime.utcnow()
```

However, doing so, I'm not sure we could still unpublish an article 🤔